index.html

```diff
@@ -11,12 +11,18 @@
 
 <body>
```

Comment:
Not positive I'm seeing formatting correctly, but it looks like your `<head>` and `<body>` tags, plus their contents, should be indented one level as they're inside the `<html>` tags.

Otherwise 👍